includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -229,300 +267,132 @@ public function setup_wizard_content() {
+			</div>
+			<div style="width:33%; float:left;">
+				<label for="store_city"><?php esc_html_e( 'City', 'woocommerce' ); ?></label>
+				<input type="text" id="store_city" name="store_city" required value="<?php echo esc_attr( $city ); ?>" />
+			</div>
+			<div style="width:33%; float:left;">
+				<label for="store_country_state"><?php esc_html_e( 'Country / State', 'woocommerce' ); ?></label>
+				<select id="store_country_state" name="store_country_state" style="width:100%;" required data-placeholder="<?php esc_attr_e( 'Choose a country&hellip;', 'woocommerce' ); ?>" aria-label="<?php esc_attr_e( 'Country', 'woocommerce' ) ?>" class="wc-enhanced-select">
+					<?php WC()->countries->country_dropdown_options( $country, $state ); ?>
+				</select>
+			</div>
+			<div style="width:33%; float:left;">
+				<label for="store_postcode"><?php esc_html_e( 'Postcode / ZIP', 'woocommerce' ); ?></label>
+				<input type="text" id="store_postcode" name="store_postcode" required value="<?php echo esc_attr( $postcode ); ?>" />
+			</div>
+			<div>
+				<label for="currency_code"><?php esc_html_e( 'Store currency', 'woocommerce' ); ?></label>
+				<select id="currency_code" name="currency_code" style="width:100%;" required data-placeholder="<?php esc_attr_e( 'Choose a currency&hellip;', 'woocommerce' ); ?>" class="wc-enhanced-select">
+					<option value=""><?php esc_html_e( 'Choose a currency&hellip;', 'woocommerce' ); ?></option>
+				<?php foreach ( get_woocommerce_currencies() as $code => $name ) : ?>
+					<option value="<?php echo esc_attr( $code ); ?>" <?php selected( $currency, $code ); ?>><?php printf( esc_html__( '%1$s (%2$s)', 'woocommerce' ), $name, get_woocommerce_currency_symbol( $code ) ); ?></option>
+				<?php endforeach; ?>
+				</select>
+			</div>
+			<div>
+				<label for="product_type"><?php esc_html_e( 'What type of product do you plan to sell?', 'woocommerce' ); ?></label>
+				<select id="product_type" name="product_type" style="width:100%;" required data-placeholder="<?php esc_attr_e( 'Please choose one&hellip;', 'woocommerce' ); ?>" class="wc-enhanced-select">
+					<option value="" <?php selected( $product_type, '' ); ?>><?php esc_html_e( 'Please choose one&hellip;', 'woocommerce' ); ?></option>
+					<option value="physical" <?php selected( $product_type, 'physical' ); ?>><?php esc_html_e( 'I plan to sell physical products', 'woocommerce' ); ?></option>
+					<option value="virtual" <?php selected( $product_type, 'virtual' ); ?>><?php esc_html_e( 'I plan to sell digital products', 'woocommerce' ); ?></option>
+					<option value="both" <?php selected( $product_type, 'both' ); ?>><?php esc_html_e( 'I plan to sell both', 'woocommerce' ); ?></option>
+				</select>
+			</div>
+			<p><?php esc_html_e( 'Transforming your site into an online store requires WooCommerce to create a few pages for you. These pages are: shop, cart, my account, and checkout.', 'woocommerce' ); ?></p>
```

Comment:
Since this can no longer be skipped I don't think this is useful?

Wouldn't a checkbox to do this be sensible? Otherwise maybe dump it?

Reply:
From a purely aesthetic standpoint I'd like to avoid adding a checkbox, though I'm curious - what's not useful about having store pages created?

Reply:
@jeffstieler The actual action of adding pages is useful. It's the message itself that is not IMO - it happens regardless. 

Previously you could skip the step.

Reply:
@mikejolley - I was thinking that the text may still be useful so that people aren't wondering where the pages came from after they're done the wizard? Could be surprising, no? cc @olaolusoga for a design perspective on this.

Reply:
Agree @v18 . It's worth keeping for the cohort that require explicit informing about the installed pages. It's also there as a solution to the initial reason for the previous onboarding step (inform people of the installed pages). Though it can't be skipped, we could technically say the previous onboading was the same, since a WooCommerce store couldn't "exist" without the pages.

Reply:
@olaolusoga In previous version, you could skip that entire step and continue on with the rest of the wizard if the pages were not needed for any reason (rare?).

It's minor but worth pointing out this is now forced, thats why I thought this text was just an extra distraction if it's happening regardless.

Reply:
Good point. Though it's forced, it does still feel like something we should inform ppl about, so they don't unintentionally delete the pages. 

This could be something we queue up for an A/B test. See if on-boarding is affected by keeping/removing it.

Reply:
@olaolusoga 👍 So, sounds like resolution is to leave off for now, and perhaps use it in an A/B test later.